qt_gui_cpp/src/qt_gui_cpp/CMakeLists.txt

```diff
@@ -34,10 +33,12 @@ set(qt_gui_cpp_LINK_LIBRARIES
 
 qt5_wrap_cpp(qt_gui_cpp_MOCS ${qt_gui_cpp_HDRS})
 
-include_directories(${PROJECT_NAME} ${qt_gui_cpp_INCLUDE_DIRECTORIES})
+ament_export_dependencies(pluginlib)
+include_directories(${PROJECT_NAME} ${qt_gui_cpp_INCLUDE_DIRECTORIES} ${pluginlib_INCLUDE_DIRS})
 add_library(${PROJECT_NAME} ${qt_gui_cpp_SRCS} ${qt_gui_cpp_MOCS})
-target_link_libraries(${PROJECT_NAME} ${qt_gui_cpp_LINK_LIBRARIES} ${catkin_LIBRARIES} Qt5::Widgets ${TinyXML_LIBRARIES})
+
+target_link_libraries(${PROJECT_NAME} ${qt_gui_cpp_LINK_LIBRARIES} ${ament_LIBRARIES} Qt5::Widgets ${TinyXML_LIBRARIES})
 
 install(TARGETS ${PROJECT_NAME}
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib)
```

Comment:
This won't work for Windows but we can keep that separate from this PR.

Reply:
Is there a recommended variable to use?

Reply:
No, but for Windows the destination is determined by `RUNTIME DESTINATION` and it goes to `bin` (see e.g. https://github.com/ros2/rcutils/blob/e21abcc5512e869a9e307216a3f27fdda0737e6b/CMakeLists.txt#L105).

For Windows support the bigger thing will be to handle symbol visibility explicitly though.